janus.c

```diff
@@ -292,8 +309,9 @@ static json_t *janus_info(const char *transaction) {
 	json_object_set_new(info, "candidates-timeout", json_integer(candidates_timeout));
 	json_object_set_new(info, "server-name", json_string(server_name ? server_name : JANUS_SERVER_NAME));
 	json_object_set_new(info, "local-ip", json_string(local_ip));
-	if(public_ip != NULL)
-		json_object_set_new(info, "public-ip", json_string(public_ip));
+	if(janus_get_public_ip_count() > 0) {
+		json_object_set_new(info, "public-ip", json_string(janus_get_public_ip(0)));
```

Comment:
I think `info` should also return the other public addresses, if configured: it's ok to keep `public-ip` as it is for backwards compatibility, but maybe there should also be an array with all the configured addresses if count > 1?

Reply:
Added and rebased.